sklearn/kernel_approximation.py

```diff
@@ -1,17 +1,19 @@
 """
 The :mod:`sklearn.kernel_approximation` module implements several
-approximate kernel feature maps base on Fourier transforms.
+approximate kernel feature maps based on Fourier transforms and Count Sketches.
 """
 
 # Author: Andreas Mueller <amueller@ais.uni-bonn.de>
-#
+#         Daniel Lopez-Sanchez (TensorSketch) <lope@usal.es>
+
 # License: BSD 3 clause
 
 import warnings
 
 import numpy as np
 import scipy.sparse as sp
 from scipy.linalg import svd
+from scipy import fftpack
```

Comment:
fftpack is legacy, it should be `scipy.fft` on newer scipy versions https://docs.scipy.org/doc/scipy/reference/fftpack.html

Though might have to still use fftpack on older scipy versions.

Reply:
I changed that, but `doc-min-dependencies` failed. What should I do?

https://app.circleci.com/pipelines/github/scikit-learn/scikit-learn/6636/workflows/80111186-c49a-4a6c-808d-6c222566c631/jobs/115998

Reply:
Sadly `scipy.fft` is a function you can import even in `scipy <= 1.3`, but the following should work:
```
try:
    from scipy.fft import fft, ifft
except ImportError:   # scipy < 1.4
    from scipy.fftpack import fft, ifft
```